### ISSUE TITLE
Pipeline for labeling training data and training networks

### ISSUE THREAD

**Issue by esteinig** [python-friends/auto-scope#2]
Pipeline of collecting images, labeling cell types / parasites (Leaflet) and training networks, how would we implement this link to other modules in Autoscope?

**Comment by Wytamma** [comment]
[This](http://leaflet.github.io/Leaflet.draw/docs/examples/full.html) might be useful for labelling data.

**Comment by esteinig** [comment]
Leaflet would be great for targeting parasites and counting / binary classification. If we want to extend to cell types or so later ,we might want to implement bounding boxes?

**Comment by Wytamma** [comment]
Yeah I think that’s a good idea. If we convert the slide scans into map tiles we can use them with leaflet (see [gdal2tiles.py](https://www.gdal.org/gdal2tiles.html). We can attach a simple database to the example above and use the leaflet polygon feature to add bounding boxes. Some simple export code could get it from the DB to a ML image data format.

**Comment by esteinig** [comment]
Yes, was thinking about the polygon feature, that should be fairly simple. Will check this out when I am out of the lab.
